collage/layout.py

```diff
@@ -0,0 +1,213 @@
+        self.width_entry = tk.Entry(entries_frame, width = bw)
+        self.width_entry.insert(0, '300')
+        
+        self.height_label = tk.Label(entries_frame, text = "Height in pixels", padx = px)
+        self.height_entry = tk.Entry(entries_frame, width = bw)
+        self.height_entry.insert(0, '300')
+        
+        self.margin_label = tk.Label(entries_frame, text = "Margin in pixels", padx = px)
+        self.margin_entry = tk.Entry(entries_frame, width = bw)
+        self.margin_entry.insert(0, '3')
+        
+        self.corner_label = tk.Label(entries_frame, text = "Corner curvature (0-1)", padx = px)
+        self.corner_entry = tk.Entry(entries_frame, width = bw)
+        self.corner_entry.insert(0, '0') 
+        
+        labels_entries = [
+            (self.width_label, self.width_entry),
+            (self.height_label, self.height_entry),
+            (self.margin_label, self.margin_entry),
+            (self.corner_label, self.corner_entry)
+        ]
+        for i, (label, entry) in enumerate(labels_entries):
+            label.grid(row = i, column = 0, sticky = 'e')
+            entry.grid(row = i, column = 1, sticky = 'w')
+        
+        # "Add text" button
+        text_button_frame = tk.Frame(left_frame, bd = 10)
+        text_button_frame.grid(row = 2, column = 0, sticky = 's')
+        self.text_button = tk.Button(text_button_frame, text = "Add text...")
+        self.text_button.grid()
+        self.text_button.bind('<Button-1>', self.open_text_window)
+        
+        # TODO: stick to all sides
+        
+        right_frame = tk.LabelFrame(self, text = 'Right', padx = 10, pady = 10)
+        right_frame.grid(row = 0, column = 1, sticky = 'sne')
+        
+        # TODO: bind commands
+        
+        # "Add photo" buttons
+        self.add_left = tk.Button(right_frame, text = '+')
+        self.add_right = tk.Button(right_frame, text = '+')
+        self.add_up = tk.Button(right_frame, text = '+')
+        self.add_down = tk.Button(right_frame, text = '+')
+        self.add_left.grid(row = 1, column = 0, sticky = 'w')
+        self.add_right.grid(row = 1, column = 2, sticky = 'w')
+        self.add_up.grid(row = 0, column = 1, sticky = 'n')
+        self.add_down.grid(row = 2, column = 1, sticky = 's')
+        
+        # Canvas with collage result
+        self.collage = tk.Canvas(right_frame)
+        self.collage.grid(row = 1, column = 1, sticky = 'nswe')
+        
+    def open_text_window(self, event):
+        # Output: tk.canvas object
+        root = tk.Tk()
+        window = TextConfigureApp(master=root)
+        window.mainloop()
+        return window.get_return()
+        
+        
+class TextConfigureApp(tk.Frame):
+    def __init__(self, master=None):
+        super().__init__(master)
+        self.master = master
+        self.rgb = (0, 0, 0)
+        self.grid()
+        self.create_widgets()
+
+    def create_widgets(self):
```

Comment:
Separate this method for:
* create_text_redactor
* create_font_chooser
* and so on